[2023-01-01T00:30:55.587+0000] {taskinstance.py:1165} INFO - Dependencies all met for <TaskInstance: dpt_cosmetic.transaction.extract manual__2023-01-01T00:30:42.749234+00:00 [queued]>
[2023-01-01T00:30:55.646+0000] {taskinstance.py:1165} INFO - Dependencies all met for <TaskInstance: dpt_cosmetic.transaction.extract manual__2023-01-01T00:30:42.749234+00:00 [queued]>
[2023-01-01T00:30:55.647+0000] {taskinstance.py:1362} INFO - 
--------------------------------------------------------------------------------
[2023-01-01T00:30:55.649+0000] {taskinstance.py:1363} INFO - Starting attempt 1 of 1
[2023-01-01T00:30:55.650+0000] {taskinstance.py:1364} INFO - 
--------------------------------------------------------------------------------
[2023-01-01T00:30:55.738+0000] {taskinstance.py:1383} INFO - Executing <Task(ExtractOperator): transaction.extract> on 2023-01-01 00:30:42.749234+00:00
[2023-01-01T00:30:55.759+0000] {standard_task_runner.py:55} INFO - Started process 675 to run task
[2023-01-01T00:30:55.782+0000] {standard_task_runner.py:82} INFO - Running: ['***', 'tasks', 'run', 'dpt_cosmetic', 'transaction.extract', 'manual__2023-01-01T00:30:42.749234+00:00', '--job-id', '106', '--raw', '--subdir', 'DAGS_FOLDER/main.py', '--cfg-path', '/tmp/tmpza7vd2wr']
[2023-01-01T00:30:55.786+0000] {standard_task_runner.py:83} INFO - Job 106: Subtask transaction.extract
[2023-01-01T00:30:56.110+0000] {task_command.py:376} INFO - Running <TaskInstance: dpt_cosmetic.transaction.extract manual__2023-01-01T00:30:42.749234+00:00 [running]> on host 6bd76effbc07
[2023-01-01T00:30:56.533+0000] {taskinstance.py:1592} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=***
AIRFLOW_CTX_DAG_ID=dpt_cosmetic
AIRFLOW_CTX_TASK_ID=transaction.extract
AIRFLOW_CTX_EXECUTION_DATE=2023-01-01T00:30:42.749234+00:00
AIRFLOW_CTX_TRY_NUMBER=1
AIRFLOW_CTX_DAG_RUN_ID=manual__2023-01-01T00:30:42.749234+00:00
[2023-01-01T00:30:56.569+0000] {base.py:71} INFO - Using connection ID 'mongodb' for task execution.
[2023-01-01T00:30:56.621+0000] {logging_mixin.py:120} INFO - <pymongo.cursor.Cursor object at 0x7f7cb7424350>
[2023-01-01T00:30:56.744+0000] {logging_mixin.py:120} INFO - []
[2023-01-01T00:30:56.747+0000] {taskinstance.py:1851} ERROR - Task failed with exception
Traceback (most recent call last):
  File "/opt/airflow/dags/utilties/extract_operator.py", line 33, in execute
    data=pd.DataFrame(list(results))
NameError: name 'results' is not defined
[2023-01-01T00:30:56.785+0000] {taskinstance.py:1406} INFO - Marking task as FAILED. dag_id=dpt_cosmetic, task_id=transaction.extract, execution_date=20230101T003042, start_date=20230101T003055, end_date=20230101T003056
[2023-01-01T00:30:56.854+0000] {standard_task_runner.py:105} ERROR - Failed to execute job 106 for task transaction.extract (name 'results' is not defined; 675)
[2023-01-01T00:30:56.918+0000] {local_task_job.py:164} INFO - Task exited with return code 1
[2023-01-01T00:30:57.004+0000] {local_task_job.py:273} INFO - 0 downstream tasks scheduled from follow-on schedule check
